package.json

```diff
@@ -13,5 +13,8 @@
     "hooks": {
       "pre-commit": "lerna run --concurrency 1 --stream precommit --since HEAD"
     }
+  },
+  "dependencies": {
+    "react-minimal-pie-chart": "^7.2.0"
```

Comment:
This should probably be installed in the frontend package, not the root package.